lib/api/bucketPut.js

```diff
@@ -37,10 +39,25 @@ export default function bucketPut(authInfo, request, locationConstraint, log,
         log.trace('invalid acl header');
         return callback(errors.InvalidArgument);
     }
+    // - AWS JS SDK sends a request with locationConstraint us-east-1
+    // if no locationConstraint provided.
```

Comment:
It is more like a note explaining why we put `us-east-1` in the `locationConstraint` section